src/components/Layout.tsx

```diff
@@ -141,7 +137,7 @@ export const ExternalLink = ({
 }) => (
   <a
     href={href}
-    className={`text-auburn underline hover:text-auburn-dark ${className}`}
+    className={`text-primary underline hover:text-canada-red-900 ${className}`}
```

Comment:
Can you use `auburn` instead of `canada-red`? They're the same colour just duplicated.